Makefile

```diff
@@ -7,40 +7,48 @@ PYFILES = $(shell ls yeat/*.py yeat/*/*.py)
 help: Makefile
 	@sed -n 's/^## //p' Makefile
 
-## test:        run automated test suite
+## test:        run only short-running automated tests
 test:
-	pytest --cov=yeat -m 'not long and not bandage and not hifi and not nano'
+	pytest --cov=yeat -m short
 
 ## testlong:    run only long-running automated tests
 testlong:
-	pytest --cov=yeat -m 'long and not bandage and not hifi'
+	pytest --cov=yeat -m long
 
-## testbandage: run only bandage required automated tests
-testbandage:
-	pytest --cov=yeat -m bandage
+## testillumina:run only Illumina short-read automated tests
+testillumina:
+	pytest --cov=yeat -m illumina
 
-## testhifi:    run only PacBio HiFi-reads automated tests
+## testhifi:    run only PacBio HiFi-read automated tests
 testhifi:
-	pytest --cov=yeat -m 'hifi and not long and not bandage'
+	pytest --cov=yeat -m hifi
 
-## testnano:    run only Oxford Nanopore-reads automated tests
+## testnano:    run only Oxford Nanopore-read automated tests
 testnano:
-	pytest --cov=yeat -m 'nano and not long and not bandage'
+	pytest --cov=yeat -m nano
+
+## testbandage: run only Bandage required automated tests
+testbandage:
+	pytest --cov=yeat -m bandage
+
+## testlinux:   run only Linux required automated tests
+testlinux:
+	pytest --cov=yeat -m linux
```

Comment:
New pytest mark for running Linux only tests. Currently, there is only one linux test in the test suite.